Magento_Cms/web/css/source/pages/_404.less

```diff
@@ -1,7 +1,14 @@
+@import 'https://fonts.googleapis.com/css?family=Dosis:300,400,700,800';
+
+//
+//  Common
+//  _____________________________________________
+
 & when (@media-common = true) {
     .cms-no-route {
         background: #1d2d44ff;
         color: @color-general-white;
+        font-family: Dosis, sans-serif;
```

Comment:
create a variable for the font usage. probably it could be reused on other pages. not to hardcode that

Reply:
Ok if I create a variable of the font family. Does that mean that I should put the url link in the default.xml so it is available everywhere and not just in in 404?

Reply:
you can use the right handle only for 404. but designers don't use custom fonts only for 1 page in real projects. if the font is used, it could be used somewhere else as well. In that case, you can use default_head_blocks.xml and include the font for all the pages